### DISCUSSION TITLE
使用Array.fill()初始化一个二维数组的小坑

### DISCUSSION THREAD

**Discussion by giscus** [ztkuaikuai/blog#1] [General]
# 使用Array.fill()初始化一个二维数组的小坑

记录一些技术、生活、阅读的感悟；是我的人类使用说明书。

http://localhost:4321/posts/%E5%88%9D%E5%A7%8B%E5%8C%96%E4%B8%80%E4%B8%AA%E4%BA%8C%E7%BB%B4%E6%95%B0%E7%BB%84

**Comment by ztkuaikuai** [comment]
**just test**

**Comment by ztkuaikuai** [comment]
我的暗黑模式呢
